Add known issue to handle stuck prebackup when upgrading (#332)
vshn/k8up: docs/modules/ROOT/pages/how-tos/upgrade.adoc

# Trigger a new backup by creating a new Backup object
kubectl create -f <your-backup-file-spec>
----

=== Known issues

- PreBackups get stuck
+
If you are using PreBackupPods and multiple `Backups` are waiting for one to complete, they may get stuck waiting for PreBackup.

> In such a case, you need to delete the oldest waiting `Backup` jobs to keep 1 object only.

Reviewer: ```suggestion
In such a case, you need to just keep the most recent `Backup` job and delete all the older ones.
```

Also: Backup _jobs_ or Backup _resources_?

Alternatively, you can delete all `Backup` jobs and just wait for the next schedule.

> That should make it easier to script if you encounter this problem in a large scale.

Reviewer: ```suggestion
That is easier to script should you encounter this problem on a large scale.
```
